src/Governance.sol

```diff
@@ -231,18 +237,34 @@ contract Governance is Multicall, UserProxyFactory, ReentrancyGuard, Ownable, IG
         UserProxy userProxy = UserProxy(payable(deriveUserProxyAddress(msg.sender)));
         require(address(userProxy).code.length != 0, "Governance: user-proxy-not-deployed");
 
-        uint88 lqtyStaked = uint88(stakingV1.stakes(address(userProxy)));
+        (
+            uint256 lqtyReceived,
+            uint256 lqtySent,
+            uint256 lusdReceived,
+            uint256 lusdSent,
+            uint256 ethReceived,
+            uint256 ethSent
+        ) = userProxy.unstake(_lqtyAmount, _doSendRewards, _recipient);
 
-        (uint256 accruedLUSD, uint256 accruedETH) = userProxy.unstake(_lqtyAmount, _doSendRewards, _recipient);
-
-        emit WithdrawLQTY(msg.sender, _lqtyAmount, accruedLUSD, accruedETH);
+        emit WithdrawLQTY(msg.sender, _recipient, lqtyReceived, lqtySent, lusdReceived, lusdSent, ethReceived, ethSent);
     }
 
     /// @inheritdoc IGovernance
-    function claimFromStakingV1(address _rewardRecipient) external returns (uint256 accruedLUSD, uint256 accruedETH) {
+    function claimFromStakingV1(address _rewardRecipient) external returns (uint256 lusdSent, uint256 ethSent) {
         address payable userProxyAddress = payable(deriveUserProxyAddress(msg.sender));
         require(userProxyAddress.code.length != 0, "Governance: user-proxy-not-deployed");
-        return UserProxy(userProxyAddress).unstake(0, true, _rewardRecipient);
+
+        uint256 lqtyReceived;
+        uint256 lqtySent;
+        uint256 lusdReceived;
+        uint256 ethReceived;
+
+        (lqtyReceived, lqtySent, lusdReceived, lusdSent, ethReceived, ethSent) =
```

Comment:
I guess lqty values will always be zero here, right?

Reply:
`lqtyReceived` yes, but `lqtySent` may be positive if someone has previously sent LQTY to the `UserProxy` address, which `UserProxy.unstake()` would send to `_rewardRecipient`. This is documented in `IGovernance.sol`.

Reply:
I considered returning the value of `lqtySent`, but in the end I didn't do it because it wouldn't normally be useful.